### PR TITLE
Create a better UI for notification page

### PR DESCRIPTION
**Pull request by Palakkgoyal**
This PR closes issue no. #636 
The badge above shows the number of notifications.
I have given styling to the name of friend-request sender. But, that will not show up in this notif as I have updated the notif doc and make it such that it will recieve the sender name seperately so that user can click on it and check their profile and the notif in the screenshot is based on old doc.
Screenshot (581)
Screenshot (582)
Screenshot 2023-07-05 000846

### CODE REVIEW

File: src/pages/Signup/index.jsx [hunk 1]
```diff
@@ -57,7 +57,7 @@ const SignupScreen = () => {
     if (!regex.test(name)) {
       setUsernameAvailable(false);
     } else {
-      debounce(findUsernameInDB);
+      debounce(findUsernameInDB());
```
Comment (narayan954) [inline src/pages/Signup/index.jsx:60]: aren't we passing function reference to the debounce function instead of actually calling the function?

**Review by narayan954 — APPROVED** [review]
looks good to me!
just one doubt i have expressed in review comment.
Merging this

**Comment by Palakkgoyal** [comment]
Yes, I did this intentionally, cause I checked that the debounce function is not calling the findUsername function. It was just a quick fix but, we need to make it correct.

**Comment by narayan954** [comment]
> Yes, I did this intentionally, cause I checked that the debounce function is not calling the findUsername function. It was just a quick fix but, we need to make it correct.

It's correct now, right?

**Pull request merged**